lopper/assists/baremetal_gentestapp_xlnx.py

```diff
@@ -128,6 +130,17 @@ def xlnx_generate_testapp(tgt_node, sdt, options):
         except KeyError:
             drv_config_name = drv_name
 
+        if drv_config_name == 'XAxiEthernet':
+           for node in node_list:
+               if "xlnx,eth-dma" in node["compatible"].value:
+                   dma_node_list.append(node)
+                   dma_label = get_label(sdt, symbol_node, node)
+                   testapp_name.update({dma_label: 'XAxiDma'})
+               if "xlnx,eth-mcdma" in node["compatible"].value:
+                   dma_node_list.append(node)
+                   dma_label = get_label(sdt, symbol_node, node)
+                   testapp_name.update({dma_label: 'XMcdma'})
+
```

Comment:
We have a axi_fifo use case as well AFAIK are we not supporting it in the driver examples?

Reply:
Currently we don't have support for axi_fifo.